oox/oox.h

```diff
@@ -161,7 +207,7 @@ struct task : public tbb_task, task_life {
         __OOX_ASSERT(false, "");
         return nullptr;
     }
-    virtual tbb_task* cancel(execution_data& ed) override {
+    TASK_CANCEL_METHOD {
```

Comment:
only TBB has this virtual cancel methods, so may be no need in name defined behind the macro.. unless we want to highlight similarity with execute method.